lib/avromatic/model/types/boolean_type.rb

```diff
@@ -33,6 +33,10 @@ def coercible?(input)
         def serialize(value, **)
           value
         end
+
+        def referenced_models
+          []
```

Comment:
It might be worth declaring an `EMPTY_ARRAY` constant in `AbstractType` that subclasses can use. I don't expect a noticeable perf gain though since this method won't be called frequently. Up to you.